sim/hunter/item_sets_pve_phase_8.go

```diff
@@ -0,0 +1,225 @@
+// 			hunter := agent.(HunterAgent).GetHunter()
+// 			hunter.applyScarletEnclaveRanged4PBonus()
+// 		},
+// 		6: func(agent core.Agent) {
+// 			hunter := agent.(HunterAgent).GetHunter()
+// 			hunter.applyScarletEnclaveRanged6PBonus()
+// 		},
+// 	},
+// })
+
+// Your Shots deal 20% increased damage to targets afflicted with your Serpent Sting.
+func (hunter *Hunter) applyScarletEnclaveRanged2PBonus() {
+	label := "S03 - Item - Scarlet Enclave - Hunter - Ranged 2P Bonus"
+	if hunter.HasAura(label) {
+		return
+	}
+
+	damageMod := hunter.AddDynamicMod(core.SpellModConfig{
+		Kind:      core.SpellMod_DamageDone_Pct,
+		ClassMask: ClassSpellMask_HunterShots,
+	})
+
+	core.MakeProcTriggerAura(&hunter.Unit, core.ProcTrigger{
+		Name:           label,
+		ClassSpellMask: ClassSpellMask_HunterShots,
+		Callback:       core.CallbackOnApplyEffects,
+		Handler: func(sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+			damageMod.UpdateFloatValue(core.TernaryFloat64(hunter.SerpentSting.Dot(result.Target).IsActive(), 1.20, 1.0))
+			damageMod.Activate()
+		},
+	})
+}
+
+// Reduces the cooldown on your Chimera Shot, Explosive Shot, and Aimed Shot abilities by 1.5 sec and reduces the cooldown on your Kill Shot ability by 3sec.
+func (hunter *Hunter) applyScarletEnclaveRanged4PBonus() {
+	label := "S03 - Item - Scarlet Enclave - Hunter - Ranged 4P Bonus"
+	if hunter.HasAura(label) {
+		return
+	}
+
+	apBonus := hunter.NewDynamicMultiplyStat(stats.AttackPower, 1.2)
+	apRangedBonus := hunter.NewDynamicMultiplyStat(stats.RangedAttackPower, 1.2)
+
+	procAura := hunter.GetOrRegisterAura(core.Aura{
+		Label:    "Wicked Shot",
+		ActionID: core.ActionID{SpellID: 1226136},
+		Duration: time.Second * 10,
+
+		OnGain: func(aura *core.Aura, sim *core.Simulation) {
+			hunter.EnableDynamicStatDep(sim, apBonus)
+			hunter.EnableDynamicStatDep(sim, apRangedBonus)
+		},
+		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+			hunter.DisableDynamicStatDep(sim, apBonus)
+			hunter.DisableDynamicStatDep(sim, apRangedBonus)
+		},
+	})
+
+	core.MakeProcTriggerAura(&hunter.Unit, core.ProcTrigger{
+		Name:     label,
+		Callback: core.CallbackOnSpellHitDealt,
+		Outcome:  core.OutcomeCrit,
+		ProcMask: core.ProcMaskRanged,
+		Handler: func(sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+			procAura.Activate(sim)
+		},
+	})
+}
+
+// Your Multi-Shot hits 2 additional targets, and your Kill Shot and Chimera Shot hits increase the damage done by your next Multi-Shot by 100%.
+func (hunter *Hunter) applyScarletEnclaveRanged6PBonus() {
+	label := "S03 - Item - Scarlet Enclave - Hunter - Ranged 6P Bonus"
+	if hunter.HasAura(label) {
+		return
+	}
+
+	damageMod := hunter.AddDynamicMod(core.SpellModConfig{
+		Kind:      core.SpellMod_DamageDone_Flat,
+		ClassMask: ClassSpellMask_HunterMultiShot,
+		IntValue:  100,
+	})
+
+	multishotAura := hunter.RegisterAura(core.Aura{
+		Label:    "Multi-Shot +100% Damage",
```

Comment:
Can we add the Action ID here so that it shows up in the UI? https://www.wowhead.com/classic-ptr/spell=1233451/trick-shots